kimchi/build.rs

```diff
@@ -1,15 +1,13 @@
 // Build script
 
-use std::process::Command;
+use cargo_spec::build::{self, OutputFormat};
 
 fn main() {
     // Rebuild specification
-    assert!(
-        Command::new("make")
-            .args(["-C", "../book/specifications/kimchi", "build"])
-            .status()
-            .expect("failed to get status")
-            .success(),
-        "failed to generate specification markdown"
-    );
+    build::build(
+        "../book/specifications/kimchi/Specification.toml".into(),
+        Some("../book/src/specs/kimchi.md".into()),
+        OutputFormat::Markdown,
+    )
+    .expect("failed to generate specification");
```

Comment:
like we discussed this is not going to work, as it will not detect changes to rerun this command (unless you use the `cargo-rerun-something:...`

Reply:
this didn't work?

```rust
let files = build::build(toml_spec, Some(output_file), OutputFormat::Markdown);
for file in files {
  println!(format!("cargo:rerun-if-changed={}", file));
}
```

Reply:
I have not added that part yet because I found that it appears to be working without it and I want to read the documentation for `cargo:rerun-if-changed` carefully.

Reply:
> Oh really? Is it that simple?

More or less, but we lose our second goal of linting markdown.

Reply:
ah interesting, I guess that makes sense. But yeah it won't work if you modify the template.md files : o so probably better to have it